src/aspire/ctf/ctf_estimator.py

```diff
@@ -1,8 +1,26 @@
 """
+Contains code supporting CTF parameter estimation.
+Generally, this is a port of ASPIRE-CTF from MATLAB.
+
+See paper:
+
+  |    "Reducing bias and variance for CTF estimation in single particle cryo-EM"
+  |    Ayelet Heimowitz, Joakim Andén, Amit Singer
+  |    Ultramicroscopy, Volume 212, 2020
+  |    https://doi.org/10.1016/j.ultramic.2020.112950.
+
+Note:
+`CtfEstimator` computes the background as a monotonically decreasing
```

Comment:
Double back-ticks if you'd rather this look like:
![Screenshot 2023-11-09 at 10 49 07 AM](https://github.com/ComputationalCryoEM/ASPIRE-Python/assets/66811911/729595da-eea3-4c36-a809-e058d06047d3)

Reply:
Right, always forget that for sphinx. Thanks will do.